deepchem/data/__init__.py

```diff
@@ -17,4 +17,3 @@
 from deepchem.data.data_loader import SDFLoader
 from deepchem.data.data_loader import FASTALoader
 from deepchem.data.data_loader import ImageLoader
-import deepchem.data.tests
```

Comment:
Currently, `find_packages(exclude=["*.tests"])` in setup.py ignores tests directory. So, I faced import error in this line when importing deepchem. I think test directory should not exported.